internal/cli/deployment_destroy.go

```diff
@@ -63,7 +63,8 @@ func (c *DeploymentDestroyCommand) Run(args []string) int {
 		for _, deployment := range deployments {
 			// Can't destroy a deployment that was not successful
 			if deployment.Status.GetState() != pb.Status_SUCCESS {
-				continue
+				c.ui.Output("Deployment %d was not successful - destroy may not completely destroy "+
+					"all resources", deployment.Sequence, terminal.WithWarningStyle())
```

Comment:
Alternatively to my other comment about continuing instead of returning the sentinel error, maybe we should move this `GetState` error handling logic into the error handling logic below. 

In my opinion the only time it should really matter that we output to the user that the deployment in question was not in a successful state should be if we attempt to destroy it and that attempt failed. If it was in a non-successful state but `Destroy()` ran successfully, then I don't think we need to output anything at all.